rules/linux/execution_shell_via_meterpreter_linux.toml

```
@@ -0,0 +1,92 @@
+[metadata]
+creation_date = "2023/08/10"
+integration = ["auditd_manager"]
+maturity = "production"
+min_stack_comments = "The sampling feature within EQL was introduced in 8.6.0"
+min_stack_version = "8.6.0"
+updated_date = "2023/08/10"
+
+[rule]
+author = ["Elastic"]
+description = """
+This detection rule identifies a sample of suspicious Linux system file reads used for system fingerprinting, leveraged 
+by the Metasploit Meterpreter shell to gather information about the target that it is executing its shell on. Detecting
+this pattern is indicative of a successful meterpreter shell connection. 
+"""
+from = "now-9m"
+index = ["auditbeat-*", "logs-auditd_manager.auditd-*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Potential Meterpreter Reverse Shell"
+note = """## Setup
+This rule requires the use of the `auditd_manager` integration. `Auditd_manager` is a tool designed to simplify and enhance the management of the audit subsystem in Linux systems. It provides a user-friendly interface and automation capabilities for configuring and monitoring system auditing through the auditd daemon. With `auditd_manager`, administrators can easily define audit rules, track system events, and generate comprehensive audit reports, improving overall security and compliance in the system. The following steps should be executed in order to install and deploy `auditd_manager` on a Linux system. 
+
+```
+Kibana -->
+Management -->
+Integrations -->
+Auditd Manager -->
+Add Auditd Manager
+```
+
+`Auditd_manager` subscribes to the kernel and receives events as they occur without any additional configuration. However, if more advanced configuration is required to detect specific behavior, audit rules can be added to the integration in either the "audit rules" configuration box or the "auditd rule files" box by specifying a file to read the audit rules from. 
+
+For this detection rule to trigger, the following additional audit rules are required to be added to the integration:
+
+```
+-w /proc/net/ -p r -k audit_proc
+-w /etc/machine-id -p wa -k machineid
+-w /etc/passwd -p wa -k passwd
+```
+
+Add the newly installed `auditd manager` to an agent policy, and deploy the agent on a Linux system from which auditd log files are desirable.
+"""
+risk_score = 47
+rule_id = "5c895b4f-9133-4e68-9e23-59902175355c"
+severity = "medium"
+tags = ["Domain: Endpoint", "OS: Linux", "Use Case: Threat Detection", "Tactic: Execution"]
```

You should add this field.

```suggestion
tags = ["Domain: Endpoint", "OS: Linux", "Use Case: Threat Detection", "Tactic: Execution"]
timestamp_override = "event.ingested"
```